### PR TITLE
Integrate products in the admin's Create Event UI

### PR DESCRIPTION
**Pull request by gldraphael**
- [x] Dynamically add products with a bootstrap modal.
- [x] POST the products with the correct `name`s.
- ~Client side validation. (Requires #32)~ See #33 
- [x] Product variant integration.
- [x] Delete product.
- [x] Delete product variant.

### CODE REVIEW

**Comment by gldraphael** [comment]
I wasn't sure if `document.querySelectorAll().forEach()` was a safe bet especially after reading [this 2016 article](https://css-tricks.com/snippets/javascript/loop-queryselectorall-matches/).

**Comment by losviko** [comment]
Sorry, havent had time to look into this this weekend. Could you continue on changing the Edit page to use the eventservice as well - and perhaps copy the products admin ui to the edit page also?

File: src/EventManagement.Web/Pages/Admin/Events/Edit.cshtml.cs [hunk 1]
```diff
@@ -46,16 +47,7 @@ public EditModel(ApplicationDbContext context)
                 return Page();
             }
 
-            _context.Attach(EventInfo).State = EntityState.Modified;
```
Comment (gldraphael) [inline src/EventManagement.Web/Pages/Admin/Events/Edit.cshtml.cs]: @losviko any specific reason you chose to use this over `_context.EventInfos.Update(EventInfo)`?

Comment (losviko) [reply src/EventManagement.Web/Pages/Admin/Events/Edit.cshtml.cs]: It seems to generate smoother sql statements with only the changed fields getting updated, but I have no feelings here :)

**Pull request merged**